MEMBERSHIP.md

```diff
@@ -68,13 +80,16 @@ Any person who meets the requirements may be nominated by a contributor, includi
 * Ensure that the sponsors are `@mentioned` on the issue.
 * Make sure that the list of contributions included is representative of the work on the project.
 
-#### Triager responsibilities and privileges
+#### Reviewer responsibilities and privileges
 
-As a project Triager, you have the following responsibilities and privileges:
+As a project Reviewer, you have the following responsibilities and privileges:
 
+* You have the permission to approve, but not merge, a PR.
```

Comment:
Can reviewer not merge a PR? Above in the permissions for Reviewer it is mentioned that they have write access with merging restricted via CODEOWNERS.

Reply:
@vikas-agarwal76 I think this could be reworded, but the `reviewer` could merge a PR (they would have write access), it would just need a `maintainer` approval first if "Require review from Code Owners" is set as a branch protection rule. Does this clarify? If so, I can alter the sections pertaining to this.

Reply:
I refine the section about to be more explicit about what would be restricted through CODEOWNERS